persistence/src/commonMain/kotlin/com/wire/kalium/persistence/backup/DatabaseImporter.kt

```diff
@@ -11,6 +11,9 @@ class DatabaseImporterImpl(private val sqlDriver: SqlDriver) : DatabaseImporter
 
     override suspend fun importFromFile(filePath: String, fromOtherClient: Boolean, userDBSecret: UserDBSecret?) {
         val isDBSQLCiphered = userDBSecret != null && userDBSecret.value.isNotEmpty()
+        sqlDriver.execute("""BEGIN""")
```

Comment:
Moving this was needed.

When WAL is enabled, the attaching to another database changes slightly and the DB attached before the transaction was not being recognised inside the transaction.

The good thing is that this code works both with or without WAL mode.